packages/react-dev-utils/paths/copyPublicFolder.js

```diff
@@ -0,0 +1,7 @@
+var fs = require('fs-extra');
+module.exports = paths => function copyPublicFolder() {
```

Comment:
I think it's fine for this one to stay in `build.js` because it's highly project-specific and folks may want to customize it further.